src/app/Table.elm

```diff
@@ -11,93 +11,114 @@ import Html exposing (..)
 import Html.Attributes exposing (..)
 import Html.Events exposing (..)
 import Types exposing (PlayerID, PlayerName)
-import TableMsg exposing (Msg(..))
 import Dice
+import Identicon exposing (identicon)
 
 
 type alias DiceCount =
     Int
 
 
 type alias TablePlayer =
-    { id : PlayerID
+    { id : Int
```

Comment:
I probably missed something, but is there a reason we reverted to the less specific `Int` type here (and for the new `currentPlayerId` state property a few lines below)?